cli/src/makeProject.ts

```diff
@@ -119,9 +119,9 @@ export async function makeProject<UserDataType = unknown>({
       const anchorName = anchorify(entity);
       if (entities.has(canonicalName)) {
         // Users should fix this, but it's not a fatal error.
-        console.warn(
+        /*console.warn(
           new Error(`duplicate entity: ${canonicalName} (${pageUri})`)
-        );
+        );*/
```

Comment:
I think we're going to wonder why this is commented out later. Maybe we should add a command line option to silence this specific error.

Reply:
Added the `enableDuplicateEntityWarning` flag.